src/commands/Tripulacion/tripulacion.js

```diff
@@ -0,0 +1,517 @@
+const { Command, Possible } = require('../../index');
+const { MessageEmbed, SnowflakeUtil } = require('discord.js');
+const CrewsConstants = require('../../lib/util/CrewsConstants.js');
```

Comment:
Si no es una clase, debería estar en pascalCase.
```suggestion
const crewsConstants = require('../../lib/util/CrewsConstants.js');
```

Reply:
Y renombra todas las ocurrencias.